### DISCUSSION TITLE
Crash OBS with latest version

### DISCUSSION THREAD

**Discussion by Ibealia** [kounoike/obs-virtualbg#79] [Q&A]
Hi everyone, when I select Virtual Background Detector, the app OBS crash so I try to select in first Virtual Background Renderer and the app is ok, but when I select in the second time Virtual Background Detector, the soft crash down U_U.
I have a Win 10 in 64x so have you some clue ?
Thank you in advance !

**Comment by OrangeHedgehog** [comment]
Just adding that when I tried using version 1.1.0 I couldn't start OBS Studio.  Copied the 1.0.1 files in and OBS at least starts, but I can't get the virtual background to work (as that's a separate issues (maybe...) I'll create a separate discussion.  May be it's just fat fingers.
